api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/ProjectHasParticipantGenerator.java

```diff
@@ -141,6 +155,37 @@ public EditConfigurationVTwo getEditConfiguration(VitroRequest vreq, HttpSession
         "?vcardName a <http://www.w3.org/2006/vcard/ns#Name> . \n" +
         "?vcardName vcard:familyName ?lastName .";
 
+    public static String getDisableVCardPrefix() {
+        return "@prefix fail_pattern: ?createVCard .\n";
+    }
+
+    public static String getDisableRealPersonPrefix() {
```

Comment:
Please remove redundancy. Also, there really should be a SPARQL builder to avoid all the excessive string literals and string concatenation.

Reply:
I would disagree with that for the same reason. Currently these methods use the same variable name, but in case someone would copy the class and try to adjust variable names it would break the form execution. So I would leave it.

Reply:
Okay, but the methods return the exact same thing no matter what class members there are. Maybe the methods should be replaced with a constant?